Dockerfile

```diff
@@ -24,24 +13,22 @@ COPY ./go.sum ./go.sum
 
 RUN ./scripts/get_deps
 
-
 RUN ./scripts/build ${app_name}
 
 
 
-FROM golang:1.17-alpine as runner
-
-# It seems I have to copy these
-# TODO: Would be good if I didn't have to copy them
-# And would be good if I could configure it in the CMD line as well
-# https://github.com/Gamma169/go-service-template/issues/1
+FROM alpine:3.15 as runner
 
-ARG app_name=foobar
-ARG service_root=/opt/service
-ARG app_root=${service_root}/${app_name}
-WORKDIR $app_root
+ARG app_name
+ENV app_name ${app_name} 
 
+# Note we are following the unix file-system standard by placing our third-party app in the /opt folder
+COPY --from=builder /go/migrations /opt/${app_name}/migrations
+COPY --from=builder /go/bin/${app_name} /opt/${app_name}/${app_name}
 
-COPY --from=builder ${app_root}/bin/${app_name} ${app_root}/bin/${app_name}
+# Need workdir because otherwise docker runs command in root, and app can't find migrations folder
+WORKDIR /opt/${app_name}
 
-CMD ["./bin/foobar"]
+# Despite CMD being in "shell form" and not "exec form," 
+# it appears that the app still responds to a SIGTERM and runs with PID=1 contrary to documentation
```

Comment:
For example, as indicated [here](https://stackoverflow.com/questions/42805750/differences-between-dockerfile-instructions-in-shell-and-exec-form), where using shell form may not result in properly propagation of signals.